packages/react/src/lib/positioner/createPopperReferenceProxy.ts

```diff
@@ -4,11 +4,7 @@ import * as React from 'react'
 import * as PopperJS from 'popper.js'
 
 class ReferenceProxy implements PopperJS.ReferenceObject {
-  ref: React.RefObject<HTMLElement>
-
-  constructor(refObject) {
-    this.ref = refObject
-  }
+  constructor(private ref: React.RefObject<HTMLElement>) {}
```

Comment:
Nit, but I am not sure about this change as I see it's TS specific stuff

[Babel REPL](https://babeljs.io/repl/#?babili=false&browsers=last%208%20versions%2C%20safari%20%3E%208%2C%20firefox%20%3E%2023%2C%20chrome%20%3E%2024%2C%20opera%20%3E%2015%2C%20not%20ie%20%3C%2011%2C%20not%20ie_mob%20%3C%3D%2011&build=&builtIns=false&spec=false&loose=false&code_lz=MYGwhgzhAECC0G8BQwD2A7CAXATgV2C1RwAocBTAMwEpEBfJBlDbaMaAXmgHIxulmmVCHIA6EKgDmJMKLDUgA&debug=false&forceAllTransforms=false&shippedProposals=false&circleciRepo=&evaluate=false&fileSize=false&timeTravel=false&sourceType=module&lineWrap=true&presets=es2015%2Creact%2Ctypescript&prettier=true&targets=Node-8.9&version=7.4.4&externalPlugins=)

Reply:
well, aren't we writing TS code? I don't understand the Babel reference it contains JS code..

Reply:
We're going to use Babel as compiler, if we can't compile this code with Babel - it's a 💣

Reply:
even if that's the case the solution is not to avoid TS features but to:
- either make compilation work with whatever compiler we use (e.g.: for babel there are plugins such as: https://babeljs.io/docs/en/babel-plugin-transform-typescript)
- or mark somehow (tslint config, lint rules, etc) what TS features we cannot use in the project (if there is strong reason).

I already see this TS feature being used in:
https://github.com/stardust-ui/react/blob/5e76653d940cd037ef9fc17931da9f4e2ac8327a/packages/react/src/lib/accessibility/FocusHandling/FocusContainer.ts#L4 

### tl;dr
I think preventing devs from using language features because our tooling might not work is wrong approach (unless that language feature has perf issues, increases bundle size or introduces problems).

Reply:
NA anymore, code was removed